edb/server/bootstrap.py

```diff
@@ -1556,12 +1556,17 @@ async def _init_stdlib(
         src_hash=src_hash,
         cache_dir=cache_dir,
     )
-    tpldbdump_package = read_data_cache(
-        tpldbdump_cache,
-        pickled=True,
-        src_hash=src_hash,
-        cache_dir=cache_dir,
-    )
+
+    backend_params = cluster.get_runtime_params()
+
+    tpldbdump_package = None
+    if backend_params.instance_params.version.major == 17:
```

Comment:
Do we already keep a set of supported PostgreSQL versions somewhere?

This number should definetely be a constant somewhere, but i don't know where.

Reply:
I don't think we do, but it should probably be in `defines.py` if anywhere?